brave/src/main/java/brave/Tracing.java

```diff
@@ -97,16 +97,25 @@ public final Clock clock(TraceContext context) {
 
   abstract public ErrorParser errorParser();
 
-  // volatile for visibility on get. writes guarded by Tracing.class
-  static volatile Tracing current = null;
+  // volatile for get visibility. writes guarded by Tracing.class. Object to ensure unloadable
```

Comment:
strictly speaking, this was unloadable, but anything that used it was not. I made this Object to be consistent across our "current" tests.